leo_erc_gazebo/scripts/pdu_node

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+import rospy
+
+from geometry_msgs.msg import Point, Quaternion, Pose
+from std_srvs.srv import Trigger, TriggerResponse
+from std_msgs.msg import Empty, UInt8
+from gazebo_msgs.srv import SpawnModel, DeleteModel, GetModelState
+from tf import TransformerROS
+
+def pdu_reset(req):
+    global probe_cnt
+
+    for i in range(1,probe_number+1):
+        probe_name = model_name + str(i)
+        delete_probe_client(
+            model_name=probe_name
+        )
+    probe_cnt = 0
+    return TriggerResponse(True, "")
+
+def pdu_callback(msg):
+    global probe_cnt
+
+    if (probe_cnt >= probe_number):
+        return
+
+    probe_cnt += 1
+    probe_name = model_name+str(probe_cnt)
+
+    log = spawn_probe_client(
+        model_name=probe_name,
+        model_xml=model,
+        robot_namespace='',
+        initial_pose=Pose(pdu_translation, Quaternion(0,0,0,1)),
+        reference_frame='leo::base_footprint'
+    )
+    print(log.status_message)
+
+    pdu_dropped_pub.publish(probe_cnt)
+
+rospy.init_node("pdu_node")
+
+probe_number = 6
+probe_cnt = 0
+model_name = "probe"
+pdu_translation = Point(-0.195,0,0.12)
+
+model = rospy.get_param("/probe_description")
+
+pdu_dropped_pub = rospy.Publisher("probe_deployment_unit/probes_dropped", UInt8, queue_size=1)
+pdu_drop_sub = rospy.Subscriber("probe_deployment_unit/drop", Empty, pdu_callback)
```

Comment:
Define the Subscriber after defining ServiceProxy which is used in the callback, to avoid race conditions.